src/main/java/teammates/ui/controller/InstructorCommentsPageData.java

```diff
@@ -146,6 +120,31 @@ public boolean isViewingDraft() {
         return isViewingDraft;
     }        
 
+    private String getRecipientNames(Set<String> recipients) {
+        StringBuilder namesStringBuilder = new StringBuilder();
+        int i = 0;
+        for (String recipient : recipients) {
+            if (i == recipients.size() - 1 && recipients.size() > 1) {
+                namesStringBuilder.append("and ");
+            }
+            StudentAttributes student = roster.getStudentForEmail(recipient);
+            if (courseId.equals(recipient)) { 
```

Comment:
This comparison is not self explanatory. Why compare a course ID to a recipient?